### PR TITLE
Fix a bug in binary, update tests.

### PR DESCRIPTION
**Pull request by matanlurey**

### CODE REVIEW

File: binary/test/binary_test.dart [hunk 1]
```diff
@@ -38,69 +75,70 @@ void main() {
...
-    test('mask should correctly mask values', () {
-      int maxTimes16 = uint32.max << 4;
-      expect(uint32.mask(maxTimes16), 0xFFFFFFF0);
+void _runIntegralTests(
+  Integral type, {
+  @required int length,
+  @required bool signed,
+}) {
+  group('$type', () {
+    test('should have a lenfth of $length', () {
```
Comment (kharland) [inline binary/test/binary_test.dart]: "lenfth"

File: binary/test/binary_test.dart [hunk 2]
```diff
@@ -21,6 +22,42 @@ void main() {
     );
   });
 
+  test('$Integral#toIterable', () {
```
Comment (kharland) [inline binary/test/binary_test.dart]: Why not 
```dart
group('$Integral', ...)
```

Comment (matanlurey) [inline binary/test/binary_test.dart]: Done.

File: binary/test/binary_test.dart (hunk 1 above)
Comment (matanlurey) [inline binary/test/binary_test.dart]: Done.

**Pull request merged**